src/Apps/Artist/Routes/WorksForSale/Components/ArtistArtworkFilters.tsx

```diff
@@ -14,13 +15,17 @@ import { PriceRangeFilter } from "Components/ArtworkFilter/ArtworkFilters/PriceR
 import { SizeFilter } from "Components/ArtworkFilter/ArtworkFilters/SizeFilter"
 import { TimePeriodFilter } from "Components/ArtworkFilter/ArtworkFilters/TimePeriodFilter"
 import { WaysToBuyFilter } from "Components/ArtworkFilter/ArtworkFilters/WaysToBuyFilter"
+import { useFeatureFlag } from "System/Hooks/useFeatureFlag"
 import { useSystemContext } from "System/Hooks/useSystemContext"
 
 type ArtistArtworkFiltersProps = {}
 
 export const ArtistArtworkFilters: React.FC<
   React.PropsWithChildren<ArtistArtworkFiltersProps>
 > = props => {
+  const enableShowOnlyFramedArtworksFilter = useFeatureFlag(
+    "onyx_only_framed_artworks_filter",
+  )
```

Comment:
Do we really need the feature flag? In case something goes wrong, we cand simply roll back the feature.

Reply:
FWIW I'm in favor of the feature flag. Gives us the ability to do more fine-grained QA (in staging, or per user) before launching. I love the velocity of Hackathon but I appreciate rolling things out intentionally like this 😄

Reply:
True, the feature flag gives more flexibility for the release (and frees us from having to follow the bug-free programming principle 😄).